internal/config/config.go

```diff
@@ -29,6 +32,7 @@ type Config struct {
 	Branch    string
 	CachePath string `yaml:"cache-path"`
 	OpenGL    string
+	Engine    string
```

Comment:
Would change it to `engine-version`

Reply:
Could you also update the template `hover.yaml`

Reply:
done.